tests/Transport/NullTransportTest.php

```diff
@@ -21,8 +20,8 @@ class NullTransportTest extends TestCase
     public function testSend()
     {
         $transport = new NullTransport();
-        $event = new Event(new Configuration());
+        $event = new Event();
 
-        $this->assertTrue($transport->send($event));
+        $this->assertNotNull($transport->send($event));
```

Comment:
You should test that the returned event ID is the same as the one of the passed event